example_async_and_test/lib/main.dart

```diff
@@ -90,7 +90,7 @@ class CounterHome extends StatelessWidget {
             // `notifyListeners` in the Model).
             ScopedModelDescendant<AbstractModel>(
               builder: (context, child, model) => Text(model.counter.toString(),
-                  style: Theme.of(context).textTheme.headline4),
+                  style: Theme.of(context)!.textTheme.headline4),
```

Comment:
Are the `!` necessary for `Theme` examples? Looks like that's non-nullable: https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/theme.dart#L119

Reply:
Just pulled em in, looks like they're safe to remove. Done so on master!